### PR TITLE
Update for language changes up to 0.58.1

### PR DESCRIPTION
**Pull request by CandleCandle**
Various changes have occurred:
* FFIs need to be 'use'd
* interfaces can not have private methods
* `ponytest` was renamed to `pony_test`.

This PR is the minimum required to make it compile and have tests pass using ponyc 0.58.1

On interfaces not having private methods, I believe the method in question (`subscriber_manager()`) is not really intended to be public, but is used by the default implementations of the other methods in the interface. I have made it public but there might be a better way to achieve the intended goal.

### CODE REVIEW

**Comment by SeanTAllen** [comment]
Any idea why the nightly `make test` didn't catch these?

**Comment by ponylang-main** [comment]
Hi @CandleCandle,

The changelog - fixed label was added to this pull request; all PRs with a changelog label need to have release notes included as part of the PR. If you haven't added release notes already, please do.

Release notes are added by creating a uniquely named file in the `.release-notes` directory. We suggest you call the file `42.md` to match the number of this pull request.

The basic format of the release notes (using markdown) should be:

```
## Title

End user description of changes, why it's important,
problems it solves etc.

If a breaking change, make sure to include 1 or more
examples what code would look like prior to this change
and how to update it to work after this change.
```

Thanks.

File: reactive_streams/test.pony [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use "ponytest"
+use "pony_test"
```
Comment (SeanTAllen) [inline reactive_streams/test.pony:1]: This change is already on main.

**Comment by SeanTAllen** [comment]
@CandleCandle all this is fine on main. I'm not sure where your change is coming in from and causing the odd diff.

**Comment by SeanTAllen** [comment]
@CandleCandle your PR is coming from a version 78 commits behind main.

**Comment by CandleCandle** [comment]
My bad, sorry for wasting your time.

**Comment by SeanTAllen** [comment]
All good

**Pull request closed without merging**